cursive-core/src/printer.rs

```diff
@@ -93,8 +94,7 @@ impl<'a, 'b> Printer<'a, 'b> {
 
     /// Clear the screen.
     ///
-    /// It will discard anything drawn before.
-    ///
+    /// It will discard anything drawn before.\
```

Comment:
In this case I think it is a separate piece of information that deserves its own paragraph - it's unrelated to "discarding anything drawn before".